src/logic/fetch-elements/fetch-items.ts

```diff
@@ -43,7 +43,7 @@ function groupItemsByURN<
 type ItemCategory = 'wearable' | 'emote'
 
 function createQueryForCategory(category: ItemCategory) {
-  return `query fetchItemsByOwner($owner: String, $idFrom: String) {
+  return `query fetchItemsByOwner($owner: String, $idFrom: ID) {
```

Comment:
LGTM, should we apply the same changes when fetching lands and names?

`src/logic/fetch-elements/fetch-lands.ts`
`src/logic/fetch-elements/fetch-names.ts`

Reply:
yeah, good idea, updated!